setup/pi/configure.sh

```diff
@@ -332,6 +368,27 @@ function configure_webhook () {
   fi
 }
 
+function configure_matrix () {
+  # remove legacy file
+  rm -f /root/.teslaCamMatrixSettings
+
+  if [ "${MATRIX_ENABLED:-false}" = "true" ]
+  then
+    log_progress "Enabling Matrix"
+    mkdir -p /root/.config/matrixcli/
+    {
+      echo "def password_eval():"
+      echo "    return '$MATRIX_PASSWORD'"
+      echo ""
+      echo "accounts=[{'server': '$MATRIX_SERVER_URL', 'username': '$MATRIX_USERNAME', 'passeval': password_eval}]"
+    } > /root/.config/matrixcli/config.py
```

Comment:
Unfortunately it seems that also with matrix-commander it would be necessary to provide user, password and homeserver through a dedicated "credentials.json" file.

Reply:
The script that is now included to replace matrixcli uses command line options only. Therefor the configuration setup has now been removed.